data/ui/preferences.blp

```diff
@@ -136,20 +122,19 @@ template $PreferencesWindow : Adw.PreferencesWindow {
               _("Lower center"), _("Upper center"), _("Center"),
             ]
           };
+        }
       }
-    }
 
       Adw.ExpanderRow figure_use_custom_style {
         title: _("Use Custom Style");
```

Comment:
It should be made clear somehow that this is to set the default style, upon launch of a new session. The style of the active window is set in Figure settings.

Perhaps "Use Custom Style by Default", or "Use Custom Style upon Launch", something like that.

Reply:
Yes, absolutely